frontend_tests/node_tests/narrow_state.js

```diff
@@ -126,12 +126,13 @@ function set_filter(operators) {
 }());
 
 (function test_set_compose_defaults() {
-    set_filter([['stream', 'Foo'], ['topic', 'Bar']]);
+    set_filter([['stream', 'Foo'], ['topic', 'Bar'], ['pm-with', 'foo@bar.com']]);
```

Comment:
You shouldn't change this filter because it doesn't make sense to be narrowed to a stream/topic AND PMs. You should add tests where the filter is only PMs.

For example, you should have just `set_filter([['pm-with', 'foo@bar.com']]);` And then test `opts` and `set_compose_defaults` as before. You can add this after L134.

Reply:
done.